astroquery/cadc/core.py

```diff
@@ -694,20 +698,35 @@ def list_async_jobs(self, verbose=None):
     def _parse_result(self, result, verbose=None):
         return result
 
+    def _parse_radius(self, radius):
```

Comment:
`astroquery.utils.commons.radius_to_unit` serves the same purpose

Reply:
Ah! ok I don't know if `radius_to_unit` will parse int/float input, but I'll use it for the str/`u.Quantity` parsing. Should I try and get rid of `_parse_radius()` entirely?

Reply:
Try it, it should.  Yes, you should try to get rid of `_parse_radius` - if `commons.radius_to_unit` doesn't do exactly what you need, it's better to modify that.  This is such a common operation that we want it to remain common across all astroquery modules.

Reply:
ok! When i run `radius_to_unit(0.1)` it errors, I'll add a little snippit to `commons.radius_to_unit` to deal with numbers and just assume that its in degrees.

Reply:
That's fine.  But actually... I think we should just require that the radius be a quantity.  I believe we have enforced that requirement on other modules.  There's generally no reason to allow a default interpretation of a number.

Reply:
On closer inspection, either approach has been allowed: https://github.com/astropy/astroquery/issues/794#issuecomment-285484553
but, why not just force qty?

Reply:
I agree! it would be clearer to just force quantity. Should str be allowed too? Also, I have a question about implementation - previously `query_region` would only accept a number as input for radius (which is just an optional argument), should I maybe parse an int/float as input for `query_region` and print a warning, just to not make breaking changes? And then for `get_images` just force either string or quantity. Does that make sense?

Reply:
Yes, a deprecation warning saying "use quantities" is a great idea.

I don't think strings should be allowed unless they can be unambiguously parsed as quantities.  So, `"1 arcsec"` would be OK because `u.Quantity('1 arcsec')` is valid.  Is that what you mean?

Reply:
Ok, sounds good! And yes, that is indeed what I mean! So `'0.5 deg'` would work, but `'0.5'` would not.